test/framework/k8s_clusters.go

```diff
@@ -234,3 +234,11 @@ func (cs *K8sClusters) GetZoneEgressEnvoyTunnel() envoy_admin.Tunnel {
 func (cs *K8sClusters) GetZoneEgressEnvoyTunnelE() (envoy_admin.Tunnel, error) {
 	panic("not supported")
 }
+
+func (cs *K8sClusters) GetZoneIngressEnvoyTunnel() envoy_admin.Tunnel {
+	panic("not supported")
+}
+
+func (cs *K8sClusters) GetZoneIngressEnvoyTunnelE() (envoy_admin.Tunnel, error) {
```

Comment:
I don't see a reason to have both methods, one with E, one without E suffix.

I know that you followed ZoneEgress for that, but IMHO this is not needed. Can you remove the E version? Also zoneegress too?